packages/jetpack-blocks/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@automattic/jetpack-blocks",
+  "version": "13.1.1",
+  "description": "Gutenberg blocks for the Jetpack WordPress plugin",
+  "main": "build/editor.js",
+  "dependencies": {
+    "@babel/polyfill": "^7.2.5",
+    "@wordpress/api-fetch": "^2.2.8",
+    "@wordpress/blob": "^2.1.0",
+    "@wordpress/blocks": "^6.0.7",
+    "@wordpress/components": "^7.0.8",
+    "@wordpress/compose": "^3.0.1",
+    "@wordpress/data": "^4.2.1",
+    "@wordpress/date": "^3.0.1",
+    "@wordpress/edit-post": "^3.1.11",
+    "@wordpress/editor": "^9.0.11",
+    "@wordpress/element": "^2.1.9",
+    "@wordpress/hooks": "^2.0.5",
+    "@wordpress/i18n": "^3.1.1",
+    "@wordpress/keycodes": "^2.0.6",
+    "@wordpress/plugins": "^2.0.11",
+    "@wordpress/token-list": "^1.1.0",
+    "@wordpress/url": "^2.3.3",
+    "classnames": "^2.2.6",
+    "cookie": "^0.3.1",
+    "email-validator": "^2.0.4",
+    "gridicons": "^3.1.1",
+    "lodash": "^4.17.11",
+    "markdown-it": "8.4.2",
+    "photon": "^2.0.1",
+    "refx": "^3.1.1",
+    "resize-observer-polyfill": "^1.5.1"
+  },
```

Comment:
I think there might be a few more dependencies that we could move over from the root `package.json` since they're only used by blocks, e.g. `mapbox-gl` (?)

Reply:
There are! I just haven't gotten to them. The linter highlights them, mostly I lean on that to discover and pull them with a `^` from the root.